### PR TITLE
fix(gke-cluster): update gke-cluster config

### PR DESCRIPTION
**Pull request by verbotenj**
Closes #154

### CODE REVIEW

File: common/defaults-gcp.yaml [hunk 1]
```diff
@@ -172,50 +199,75 @@ managed_node_groups:
...
       - key: demeter.run/compute-arch
         value: "x86"
         effect: NO_SCHEDULE
     availability_zones: us-central1-a
 
-  - name: be-gp-arm64-az1
+  - name: be-di-arm64-az1
     labels:
       demeter.run/availability-sla: best-effort
       demeter.run/compute-profile: general-purpose
```
Comment (wolf31o2) [inline common/defaults-gcp.yaml:254]: The sla/profile here doesn't match the taints below.

File: common/defaults-gcp.yaml [hunk 2]
```diff
@@ -172,50 +199,75 @@ managed_node_groups:
...
+      - key: demeter.run/compute-arch
+        value: "arm64"
+        effect: NO_SCHEDULE
+    availability_zones: us-central1-b
+
+  # Disk Intensive
+  - name: be-di-x86-az1
     labels:
       demeter.run/availability-sla: best-effort
       demeter.run/compute-profile: general-purpose
```
Comment (wolf31o2) [inline common/defaults-gcp.yaml:229]: It's best-effort and general-purpose here, but consistent and disk-intensive below.

File: modules/common/cert-manager/stage2/main.tf [hunk 3]
```diff
@@ -63,7 +63,7 @@ resource "helm_release" "cert-manager" {
 
   set {
     name  = "tolerations[2].value"
-    value = "admin"
+    value = "consistent"
```
Comment (wolf31o2) [inline modules/common/cert-manager/stage2/main.tf:66]: Why are we changing these?

File: stage2/main.tf [hunk 4]
```diff
@@ -43,8 +43,9 @@ module "cert_manager" {
 # }
 
 module "o11y" {
-  source    = "../modules/common/o11y/stage2"
-  namespace = var.dmtr_namespace
+  source        = "../modules/common/o11y/stage2"
+  namespace     = var.dmtr_namespace
+  storage_class = "hyperdisk-balanced"
```
Comment (wolf31o2) [inline stage2/main.tf:48]: This needs to be `gp` on AWS and k3d.

File: stage3/kupo.tf [hunk 5]
```diff
@@ -66,8 +65,8 @@ module "ext_cardano_kupo" {
   cells = {
     "cell1" = {
       pvc = {
-        storage_size       = "10Gi"
-        storage_class_name = "gp-immediate"
+        storage_size       = "50Gi"
+        storage_class_name = "hyperdisk-balanced-immediate"
```
Comment (wolf31o2) [inline stage3/kupo.tf:69]: This needs to be configurable.

File: stage3/nodes.tf [hunk 6]
```diff
@@ -21,19 +19,19 @@ module "ext_cardano_node_crds" {
 # }
 
 module "ext_cardano_node" {
-  source                          = "git::https://github.com/blinklabs-io/demeter-ext-cardano-node.git//bootstrap?ref=feat/gcp-cloud-provider"
+  source                          = "git::https://github.com/blinklabs-io/demeter-ext-cardano-node.git//bootstrap?ref=feat/list-tolerations"
```
Comment (wolf31o2) [inline stage3/nodes.tf:22]: This can be switched back to upstream.

File: stage3/nodes.tf [hunk 7]
```diff
@@ -57,29 +55,70 @@ module "ext_cardano_node" {
     #   topology_zone = "us-central1-a"
     #   node_resources = {
     #     limits = {
-    #       "memory" = "16Gi"
-    #       "cpu"    = "8"
+    #       "memory" = "4Gi"
```
Comment (wolf31o2) [inline stage3/nodes.tf:58]: 24Gi is what I use with our helm chart.

File: stage3/nodes.tf [hunk 8]
```diff
@@ -57,29 +55,70 @@ module "ext_cardano_node" {
...
     #     requests = {
-    #       "memory" = "16Gi"
+    #       "memory" = "4Gi"
     #       "cpu"    = "2"
     #     }
     #   }
-    #   storage_size       = "200Gi"
-    #   storage_class_name = "gp"
+    #   storage_size       = "500Gi"
+    #   storage_class_name = "pd-ssd"
```
Comment (wolf31o2) [inline stage3/nodes.tf:67]: This will need to be configurable when we uncomment.

File: stage3/nodes.tf [hunk 9]
```diff
@@ -57,29 +55,70 @@ module "ext_cardano_node" {
...
+          "memory" = "2Gi"
+          "cpu"    = "8"
+        }
+        requests = {
+          "memory" = "2Gi"
+          "cpu"    = "100m"
+        }
+      }
+      storage_size       = "50Gi"
+      storage_class_name = "hyperdisk-balanced"
```
Comment (wolf31o2) [inline stage3/nodes.tf:100]: Make this configurable

**Review by wolf31o2 — CHANGES_REQUESTED** [review]

File: modules/common/cert-manager/stage2/main.tf (hunk 3 above)
Comment (verbotenj) [reply modules/common/cert-manager/stage2/main.tf:66]: We shouldn't it was temporarily.

File: common/defaults-gcp.yaml [hunk 10]
```diff
@@ -172,50 +199,75 @@ managed_node_groups:
...
+      - key: demeter.run/compute-profile
+        value: "general-purpose"
+        effect: NO_SCHEDULE
+      - key: demeter.run/compute-arch
+        value: "arm64"
+        effect: NO_SCHEDULE
+    availability_zones: us-central1-b
+
+  # Disk Intensive
+  - name: be-di-x86-az2
```
Comment (wolf31o2) [inline common/defaults-gcp.yaml:226]: Also consistent

**Review by wolf31o2 — CHANGES_REQUESTED** [review]

**Review by wolf31o2 — APPROVED** [review]

**Pull request merged**